cmd/prepare-storage.go

```diff
@@ -132,13 +132,14 @@ func prepForInitXL(firstDisk bool, sErrs []error, diskCount int) InitActions {
 	}
 
 	quorum := diskCount/2 + 1
+	readQuorum := diskCount / 2
 	disksOffline := errMap[errDiskNotFound]
 	disksFormatted := errMap[nil]
 	disksUnformatted := errMap[errUnformattedDisk]
 	disksCorrupted := errMap[errCorruptedFormat]
 
 	// No Quorum lots of offline disks, wait for quorum.
-	if disksOffline >= quorum {
+	if disksOffline > readQuorum {
```

Comment:
Not related to this commit. But when deciding on quorum we have to consider online-disks rather than offline-disks.

Because when data != parity this comparison will be incorrect.

For ex. consider 6-data and 2-parity. readQuorum will be 6. If 3 disks are down, the statement becomes:
```
if 3 > 6 {
    return WaitForQuorum
}
```
Which is incorrect because we should have returned WaitForQuorum.

If we talk in terms of online-disks, it will be like this (which is also more readable):
```
if disksOnline < readQuorum {
    return WaitForQuorum
}
```
which will work as intended.

If/when we make data/parity configurable we will have to fix these things.